docs/tutorial/vertex_cover.ipynb

```diff
@@ -0,0 +1,674 @@
+      "text/plain": [
+       "<Figure size 640x480 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "G = nx.Graph()\n",
+    "G.add_nodes_from([0, 1, 2, 3, 4])\n",
+    "G.add_edges_from([(0, 1), (0, 2), (1, 2), (1, 3), (1, 4)])\n",
+    "positions = nx.spring_layout(G)\n",
+    "color = [\"#2696EB\"] * 5\n",
+    "color[0] = color[1] = \"#EA9b26\"\n",
+    "nx.draw(G, node_color=color, pos=positions, with_labels=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "fulDayMynzrH"
+   },
+   "source": [
+    "## Formulation of the Vertex Cover Problem\n",
```

Comment:
Tex in this section is broken in local build

Reply:
Fixed by spliting the cell to several.